docs/source/installation.md

```diff
@@ -73,21 +73,46 @@ It should download a pretrained model then print something like
 
 ## Installing from source
 
-To install from source, clone the repository and install with the following commands:
+Note that the following will will install not the latest released version, but the bleeding edge `master` version, which may be not-quite ready for normal usage, but you may want to use it in case a bug has been fixed since the last official release and a new release hasn't yet been rolled out.
```

Comment:
I don't agree with `"which may be not-quite ready for normal usage"` -> master should totally be ready for "normal" usage. We very very rarely merge PRs that have crashed a "normal" usage behavior. We do merge PRs related to new functionalities which might not yet be fully ready, but this won't be different for the next release

Reply:
Rewritten - please have a look at the new incarnation